<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditProperties</class>
 <widget class="QDialog" name="DialogEditProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="boxName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineName">
        <property name="maxLength">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblAlias">
          <property name="toolTip">
           <string>Alias is only used for simlinks</string>
          </property>
          <property name="text">
           <string>Alias</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineAlias">
          <property name="maxLength">
           <number>159</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxPermissions">
     <property name="title">
      <string>Permissions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="boxPermissionUser">
        <property name="title">
         <string>User</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkUserRead">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkUserWrite">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkUserExecute">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="boxPermissionGroup">
        <property name="title">
         <string>Group</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="chkGroupRead">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkGroupWrite">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkGroupExecute">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="boxPermissionAll">
        <property name="title">
         <string>All</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="chkAllRead">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkAllWrite">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkAllExecute">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="boxPermissionSpecial">
        <property name="title">
         <string>Special</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="chkSpecialSetUid">
           <property name="text">
            <string>Set UID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSpecialSetGid">
           <property name="text">
            <string>Set GID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSpecialSticky">
           <property name="text">
            <string>Sticky</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxOwner">
     <property name="title">
      <string>Owners</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="boxOwnerGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblOwnerUser">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="boxOwnerUser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblUserGroup">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblValueUser">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineOwnerUser"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblValueGroup">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineOwnerGroup"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
